sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveQlSuite.scala

```diff
@@ -201,6 +202,31 @@ class HiveQlSuite extends SparkFunSuite with BeforeAndAfterAll {
     assert(plan.children.head.asInstanceOf[Generate].generator.isInstanceOf[JsonTuple])
   }
 
+  test("transform query spec") {
+    val plan1 = parser.parsePlan("select transform(a, b) using 'func' as c, d from e")
```

Comment:
: ) Let me import all the dsl from Catalyst. I have no clue why the test cases in sql/hive does not do it.